src/cunumeric/cunumeric_c.h

```diff
@@ -206,6 +207,58 @@ enum CuNumericBounds {
   CUNUMERIC_MAX_TASKS   = 1048576,
 };
 
+// Match these to BitGeneratorOperation in config.py
+enum CuNumericBitGeneratorOperation {
+  CUNUMERIC_BITGENOP_CREATE       = 1,
+  CUNUMERIC_BITGENOP_DESTROY      = 2,
+  CUNUMERIC_BITGENOP_RAND_RAW     = 3,
+  CUNUMERIC_BITGENOP_DISTRIBUTION = 4,
+};
+
+// Match these to BitGeneratorType in config.py
+enum CuNumericBitGeneratorType {
+  CUNUMERIC_BITGENTYPE_DEFAULT       = 0,
+  CUNUMERIC_BITGENTYPE_XORWOW        = 1,
+  CUNUMERIC_BITGENTYPE_MRG32K3A      = 2,
+  CUNUMERIC_BITGENTYPE_MTGP32        = 3,
+  CUNUMERIC_BITGENTYPE_MT19937       = 4,
+  CUNUMERIC_BITGENTYPE_PHILOX4_32_10 = 5,
+};
+
+// Match these to BitGeneratorDistribution in config.py
+enum CuNumericBitGeneratorDistribution {
+  CUNUMERIC_BITGENDIST_INTEGERS_32    = 1,
```

Comment:
are we expecting to add more distributions? if so, let's leave out enum values except the first one, otherwise you would need to renumber them every time you add a new distribution.

Reply:
Yes, same point as above. Can we push that to next PR ?